Sources/TiqrCoreObjC/Resources/General/Config.plist

```diff
@@ -26,7 +26,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>1.0</string>
```

Comment:
This seems unintentional.

Reply:
this is the package version, $(MARKETING_VERSION) does not work